social-providers/android-profile-google/src/com/soomla/profile/social/google/SoomlaGooglePlus.java

```diff
@@ -81,12 +89,14 @@
     public static final int ACTION_PUBLISH_STATUS_DIALOG = 4;
 
     private boolean autoLogin;
+    private boolean enableGameServices;
 
     private String lastContactCursor = null;
+    private HashMap<String, LeaderboardScoreBuffer> scoresCursors = null;
```

Comment:
@EugeneButusov, you should not use an exact class here, change to `Map<String, LeaderboardScoreBuffer>`